modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+    module->moveHoverCircle(i);
+        
+    return 0;
+}
+
+int disableHoverCircle(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::disableHoverCircle");
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    module->disableHoverCircle();
+        
+    return 0;
+}
+
+int lockTarget(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::lockTarget");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    if (module->getPair(id)) {
+        module->getPair(id)->lock();
+    }
+    return 0;
+}
+
+int unlockTarget(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::unlockTarget");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    if (module->getPair(id)) {
+        module->getPair(id)->unlock();
+    }
+    return 0;
+}
+
+int setImageLayerOrder(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setImageLayerOrder");
+    auto [id, i, order] = ghoul::lua::values<std::string, int, int>(L);
+
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if (module->getPair(id)) {
+        module->getPair(id)->setImageOrder(i, order);
+    }
+    return 0;
+}
+
+int loadImagesToWWT(lua_State* L) {
+    // Load images from url
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::loadImagesToWWT");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    LINFO("Connection established to WorldWide Telescope application in " + id);
+    LINFO("Loading image collections to " + id);
+
+    // Load the collections here because here we know that the browser can execute 
+    // javascript
+    std::string root = "https://raw.githubusercontent.com/WorldWideTelescope/"
+                        "wwt-web-client/master/assets/webclient-explore-root.wtml";
+
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if (module->getPair(id)) {
+        module->getPair(id)->hideChromeInterface(true);
+        module->getPair(id)->loadImageCollection(root);
+    }
+
+    return 0;
+}
+
+int startSetup(lua_State* L) {
+    // This is called when the sky_browser website is connected to OpenSpace
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::startSetup");
+
+    // Set all border colors to the border color in the master node
+    if (global::windowDelegate->isMaster()) {
+        SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+        std::vector<std::unique_ptr<TargetBrowserPair>>& pairs = module->getPairs();
+        for (std::unique_ptr<TargetBrowserPair>& pair : pairs) {
+            std::string id = pair->browserId();
+            glm::ivec3 color = pair->borderColor();
+            std::string script = fmt::format(
```

Comment:
We're not as strict on line-breaking every argument when logging using fmt format. To make it easier to read: 
```
std::string script = fmt::format(
    "openspace.skybrowser.setBorderColor('{}', {}, {}, {})", 
     id, color.r, color.g,  color.b
);
```